cocotb/log.py

```diff
@@ -125,6 +125,12 @@ def SimLog(name, ident=None):
 class SimLogFormatter(logging.Formatter):
     """Log formatter to provide consistent log message handling."""
 
+    # Removes the arguments from the base class. Docstring needed to make
+    # sphinx happy.
+    def __init__(self):
+        """ Takes no arguments. """
+        super().__init__()
```

Comment:
@imphil, if you make the change I suggested in #1410, we could add the `style` argument here.

Reply:
Nevermind, ignore that remark, I'm incorrect.